robot-logic/src/main/java/logic/Main.java

```diff
@@ -0,0 +1,31 @@
+package logic;
+
+import logic.controller.MainController;
+import lombok.extern.slf4j.Slf4j;
+
+/**
+ * Entry point...
+ */
+@Slf4j
+public class Main {
+    private final MainController mainController = new MainController();
+
+    private void launch() {
+        try {
+            mainController.launch();
+        } catch (InterruptedException e) {
+            log.debug("Interrupted exception found", e);
```

Comment:
Why `debug` level? This is exceptional case. Just one log in `error` level would be enough.